hack/run-locally.sh

```diff
@@ -111,7 +171,22 @@ if [[ -e "${CLUSTER_DIR}/manifests/cvo-overrides.yaml" ]]; then
     override_install_manifests
 fi
 
+if from_scratch ; then
+    ${INSTALLER_DIR}/bin/openshift-install --dir=${CLUSTER_DIR} create cluster &
+    ID=$!
+    echo "Started cluster installer PID ${ID}"
+fi
+
 wait_for_cluster
-setup_operator_env
+
+if [[ "${NETWORK_PLUGIN}" == "OVNKubernetes" ]]; then
+    echo "Attempting to open OVN ports for AWS..."
+    while ! hack/open-ovn-ports.sh >& /dev/null; do
```

Comment:
You don't need to retry this. The security groups have definitely been created after `wait_for_cluster` returns.